src/gleam/list.gleam

```diff
@@ -1565,6 +1565,54 @@ pub fn each(list: List(a), f: fn(a) -> b) -> Nil {
   }
 }
 
+/// Used to iterate through a list of items and execute a side effect per item as long as the
+/// previous side effect execution returned an `Ok(_)`.
+///
+/// Takes a function that returns a `Result`. The function is applied to each list item.
+/// If the function application on an item returns `Ok(_)`, then `try_each` will go on and apply the
+/// function to the next item.
+///
+/// However, if any application returns an `Error(_)`, at that point the function will stop executing.
+/// In any case the function returns `Nil`, even if it passes through all items or incurs in an
```

Comment:
I'd add a new line before this. The However sentence can stay as a separate paragraph or follow directly after "the next item. `